### ISSUE TITLE
Create documentation template 

### ISSUE THREAD

**Issue by andreaskuster** [andreaskuster/uw-nlp#5]
Well before the version 1 deadline, it would be useful for your group to put together some sort of template which has fields for all of the information they want to record from every experiment. (This could be in the form of a spreadsheet, or a rough placeholder draft with information that you fill in as you go, etc.) You should design this so that once it’s filled out for all of your proposed experiments, you will meet the overall requirements for the project (described at the end of the syllabus). For example, a group which is going to show training curves as their additional experiment would want a template which has fields for at least:
training curves showing dev evaluations every X training steps
learning rate, batch size, dropout, size of their model, etc. (they shouldn’t have “etc.” here, they should be very specific)
clear connection showing which hypothesis this experiment will support
info on the data
computational requirements
(In other words, the list that you construct should consist of the items from the numbered list in the project report description that are specific to your project, with one template filled out for each experiment you run, including baselines.)
